R/cloudRESTClientMinimal.R

```diff
@@ -0,0 +1,62 @@
+library(R6)
+library(httr)
+library(jsonlite)
```

Comment:
Package code in `R/` never has `library()` calls as this resolved in `DESCRIPTION` (and `NAMESPACE`).

If the file was meant as a script it can have the library call but needs to be in a different directory as `R/*` becomes part of the package code.